moto/config/responses.py

```diff
@@ -159,3 +159,46 @@ def put_evaluations(self):
             self._get_param("TestMode"),
         )
         return json.dumps(evaluations)
+
+    def put_organization_conformance_pack(self):
+        conformance_pack = self.config_backend.put_organization_conformance_pack(
+            self.region,
+            self._get_param("OrganizationConformancePackName"),
```

Comment:
Can we use named arguments here? Would reduce the chance of introducing an error due to the arguments being out of order

Reply:
done